src/logger.js

```diff
@@ -28,7 +35,13 @@ export default new class VueSocketIOLogger {
 
     event(text, data = ''){
 
-        if(this.debug) window.console.info(this.prefix+`%c${text}`, 'color: blue; font-weight: 600', 'color: #333333', data);
+        if(this.debug) {
+            if (this.darkMode) {
```

Comment:
better to use `window.matchMedia('(prefers-color-scheme: dark)')`

Reply:
Sure, I will add it in.
But for my case, I use light theme of os, and custom dark theme for devtool.

I think default to matchmedia and can passing value to overwrite will be better.

I'll do this and update.